public/controllers/agent/components/register-agent.js

```diff
@@ -309,7 +271,7 @@ export const RegisterAgent = withErrorBoundary(
         deployment += `WAZUH_REGISTRATION_PASSWORD='${this.state.wazuhPassword}' `;
       }
 
-      if (this.state.udpProtocol) {
+      if (!this.state.udpProtocol == true) {
```

Comment:
**nitpick**: is this comparison intentional? The `this.state.udpProtocol is negated.

Reply:
This issue was taken to another [PR](https://github.com/wazuh/wazuh-kibana-app/pull/4760), and the development has been changed.